Tests/SwiftyJSONTests/BaseTests.swift

```diff
@@ -51,11 +51,21 @@ class BaseTests: XCTestCase {
     override func setUp() {
 
         super.setUp()
+        
+        let sourceFileName = NSString(string: #file)
+        let resourceFilePrefixRange: NSRange
+        let lastSlash = sourceFileName.range(of: "/", options: .backwards)
+        if  lastSlash.location != NSNotFound {
+            resourceFilePrefixRange = NSMakeRange(0, lastSlash.location+1)
+        } else {
+            resourceFilePrefixRange = NSMakeRange(0, sourceFileName.length)
+        }
+        let pathPrefix = sourceFileName.substring(with: resourceFilePrefixRange)
```

Comment:
I use another way to construct the path in my Xcode unit test:
`URL(fileURLWithPath: #file).appendingPathComponent("../../../TestResources/default.json").standardized`

This should work on both macOS and Linux, and is easier to understand.